dev/vcpkg/setup-build-depends.sh

```diff
@@ -99,9 +99,9 @@ install_centos_7() {
     # Requires git >= 2.7.4
     if [[ "$(git --version)" != "git version 2."* ]]; then
         yum -y remove git
-        # Requires 'centos-release-scl' package to be installed.
-        yum -y install rh-git227
-        source /opt/rh/rh-git227/enable
```

Comment:
We already installed git 2.27.0, which can satisfy the minimum requirement 2.7.4. But according to the CI failure, this new git version is not found in later execution. 

The root cause is the above command doesn't change the environment for other subshell execution like `bash xxx.sh`. Maybe, we can simply add the following command to persist the version changes.

```
echo "source /opt/rh/rh-git227/enable" >> ~/.bashrc
```

This git comes from `centos-release-scl`, which should be more stable, meaning it should be always available for future use.

Reply:
@PHILO-HE Thanks. Updated to use rh-git227
 it looks like we need to enable the rh-git227 in the docker file instead in the setup script.